doc/classes/SubViewport.xml

```diff
@@ -6,6 +6,7 @@
 	<description>
 		[SubViewport] Isolates a rectangular region of a scene to be displayed independently. This can be used, for example, to display UI in 3D space.
 		[b]Note:[/b] [SubViewport] is a [Viewport] that isn't a [Window], i.e. it doesn't draw anything by itself. To display anything, [SubViewport] must have a non-zero size and be either put inside a [SubViewportContainer] or assigned to a [ViewportTexture].
+		[b]Note:[/b] [InputEvent]s are not passed to a standalone [SubViewport] by default. To ensure [InputEvent] propagation, the [SubViewport] must be inside of a [SubViewportContainer].
```

Comment:
The word "must" is too strong. There are alternatives, as for example shown in the [GUI in 3D Demo](https://github.com/godotengine/godot-demo-projects/tree/master/viewport/gui_in_3d).